tree/dataframe/src/RDFHelpers.cxx

```diff
@@ -258,14 +257,46 @@ void ProgressHelper::PrintStats(std::ostream &stream, std::size_t currentEventCo
       std::chrono::seconds remainingSeconds(
          static_cast<long long>((ComputeNEventsSoFar() - currentEventCount) / evtpersec));
       stream << " " << remainingSeconds << " "
-             << " remaining time (per file)";
+             << " remaining time (per file being processed)";
       if (fUseShellColours)
          stream << "\033[0m";
    }
 
    stream << "]   ";
 }
 
+void ProgressHelper::PrintStatsFinal(std::ostream &stream, std::size_t currentEventCount,
+                                     std::chrono::seconds elapsedSeconds) const
+{
+   auto evtpersec = EvtPerSec();
+   auto totalEvents = ComputeNEventsSoFar();
+   auto totalFiles = fTotalFiles;
+
+   std::chrono::seconds remainingSeconds(static_cast<long long>((totalEvents - currentEventCount) / evtpersec));
```

Comment:
It looks like this variable is not used in the function
```suggestion
```

Reply:
true, thanks!